scripts/entity/tactical/humans/rf_herald.nut

```diff
@@ -39,7 +39,6 @@ this.rf_herald <- ::inherit("scripts/entity/tactical/human" {
 		this.setAppearance();
 		this.getSprite("socket").setBrush("bust_base_military");
 
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
```

Comment:
Why remove Rotation? (btw the standard_bearer has rotation too).

Reply:
I don't want every Tom, Dick and Harry having rotation anymore as I find it minimizes tactical decision-making for the player. For the Herald, I find it proper that their bodyguards handle positioning and keeping their ward away from enemies. I would therefore want to remove rotation from standard bearers too.

Reply:
Sounds good. Let's remove from both.